client/src/components/ActionButton/index.tsx

```diff
@@ -1,25 +1,25 @@
-import { ActionButtonProps } from "./types";
+import { ActionButtonProps } from './types';
 
-const ActionButton = ({
+function ActionButton({
```

Comment:
`export default function ActionButton`

, then remove the last line default export